general/standards/embedded-coding-standard.rst

```diff
@@ -1,2 +1,46 @@
 Embedded (Firmware) Coding Standard
-===================================
+***********************************
```

Comment:
Does all embedded code have to be in C? The others were playing with the idea of C and I'd rather this be documented than me linking that one thread every time.